src/components/AddressAvatar.tsx

```diff
@@ -84,7 +96,27 @@ const AddressAvatar = forwardRef<HTMLDivElement, AddressAvatarProps>(
                 className='relative rounded-full'
                 fill
                 src={resolveEnsAvatarSrc(ensName)}
-                onError={onImageError}
+                onError={onEnsImageError}
+                alt='avatar'
+              />
+            </div>
+          </div>
+        )}
+
+        {profileAvatar && (
```

Comment:
Do you render 2 avatar?

Maybe better do similar w/ name, ENS | Subsocial | random

Reply:
yes, its because I want it to display the random first, if the ens or profile is loading

do you mean adding additional props for the ENS | Subsocial | random?

Reply:
but I've updated it so that it only renders ens/profile one at a time (excluding the random image),

because I think if the `Name` displays ens, it won't make sense to render subsocial profile image instead if he don't have ens avatar

Reply:
yes aggree